features/datachoices/src/main/java/org/opennms/features/datachoices/internal/UsageStatisticsReportDTO.java

```diff
@@ -80,6 +80,8 @@ public class UsageStatisticsReportDTO {
     private Map<String, Long> m_provisiondRequisitionSchemeCount;
     private Map<String, Boolean> m_services;
 
+    private int m_businessEdges;
```

Comment:
I would suggest to rename businessEdges to m_businessEdgesCount otherwise looks good!